src/validation/ValidationSchema.js

```diff
@@ -27,27 +27,22 @@ const validationSchema = yup.object({
       })
       /* eslint-disable func-names */
       .test('test', 'Minimum must be less than the maximum', function (value) {
-        const { type, maxAmount } = this.parent;
-        if (
-          type === AmountType.Range &&
-          value > 0 &&
-          maxAmount > 0 &&
-          value >= maxAmount
-        )
+        const { type, max } = this.parent;
+        if (type === AmountType.Range && value > 0 && max > 0 && value >= max)
           return false;
         return true;
       })
       .test(
         'test',
         'Range amount must have a minimum and/or a maximum',
         function (value) {
-          const { type, maxAmount } = this.parent;
-          if (type === AmountType.Range && value === 0 && maxAmount === 0)
+          const { type, max } = this.parent;
+          if (type === AmountType.Range && value === 0 && max === 0)
             return false;
           return true;
         }
       ),
-    maxAmount: yup.number().notRequired(),
+    max: yup.number().notRequired(),
```

Comment:
This needs to be at least 0 btw. It doesn't seem to give an error if I put in an negative number

Reply:
Did the field allow you to put negative numbers? For me it doesn't let me input the `-` symbol, therefore I can't input a negative number. Regardless I put the test to check for positive numbers.

Reply:
Yeah the field had allowed me to put a negative number. Looks like it's not the case anymore though